Dockerfile

```diff
@@ -10,6 +12,10 @@ ARG TOOLCHAIN=stable
 #
 # ALSO UPDATE hooks/build!
 ARG OPENSSL_VERSION=1.1.1m
+# Needs evaluation;
+# fails for libpq with:
+# "configure: error: library 'crypto' is required for OpenSSL"
+# ARG OPENSSL_VERSION=3.0.0
```

Comment:
Any support of OpenSSL 3.0.0 should happen in a separate PR, and it should wait until all the examples in the test suite  support dynamic linking of OpenSSL.

Reply:
We'll also need to consider how many existing users this breaks in the short term. We try to avoid breaking working CI setups, though in a 6-year-old project, that occasionally happens to people who pin to `:stable` instead of a version.